### ISSUE TITLE
githublogger.rb exception

### ISSUE THREAD

**Issue by shazron** [ttscoff/Slogger#403]
Seems like the activity feed has no key called "repository" anymore, it's "repo", and that key has no "owner" value anymore either.

```
          when "WatchEvent"
            if action['payload']['action'] == "started"
              output += "* Started watching [#{action['repo']['name']}](#{action['repo']['url']})\n"
              output += "    * #{action['repo']['description'].gsub(/\n/," ")}\n" unless action['repo']['description'].nil?
            end
```

patch for [these lines](https://github.com/ttscoff/Slogger/blob/b96b9c69cfdee59c0880a86c99ba84dfa0cf0e9b/plugins/githublogger.rb#L71-L75)

Using the feed in the .rb file, see an example WatchEvent [here](https://api.github.com/users/shazron/events)

**Comment by ttscoff** [comment]
I updated the logger, appears to run without error. Pushed to the develop branch, if you want to test.

**Comment by bitboxer** [comment]
This should be fixed now with #407
